src/tests/sorting.spec.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.beforeEach(async ({ loginPage, inventoryPage }) => {
+    await loginPage.navigate();
+    await loginPage.performLogin('standard_user', 'secret_sauce');
+
+    await expect(inventoryPage.headerTitle).toBeVisible();
```

Comment:
It is not a good practice to add some assertions (expect) in hooks (beforeEach). It'd be better to move these assertions to the test.